lib/galaxy/schema/schema.py

```diff
@@ -116,6 +116,15 @@ class DatasetCollectionPopulatedState(str, Enum):
     FAILED = "failed"  # some problem populating state, won't be populated
 
 
+class HashFunctionNames(str, Enum):
```

Comment:
This has to be duplicated for now since I couldn't find any common place to put the definition. The other enum is in `lib/galaxy/util/hash_util.py`